### PR TITLE
Updated deployment scripts and added documentation.

### PR DESCRIPTION
**Pull request by steegecs**

### CODE REVIEW

File: .github/workflows/deployment.yml [hunk 1]
```diff
@@ -19,4 +19,4 @@ jobs:
           HOSTED_SERVICE_ACCESS_TOKEN: ${{ secrets.HOSTED_SERVICE_ACCESS_TOKEN }}
           CHANGED_FILES: "${{ steps.files.outputs.all }}"
           ABSOLUTE_PATH: ${{ github.workspace }}
-          GRAPH_DEPLOYMENT_LOCATION: messari
+          GRAPH_DEPLOYMENT_LOCATION: steegecs
```
Comment (this-username-is-taken) [inline .github/workflows/deployment.yml:22]: change back?

File: .github/workflows/test.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+name: Deployment
```
Comment (this-username-is-taken) [inline .github/workflows/test.yaml:1]: remove?

**Comment by this-username-is-taken** [comment]
Great work!

File: .github/workflows/deployment.yml (hunk 1 above)
Comment (steegecs) [reply .github/workflows/deployment.yml]: Good catch. Changed back.

File: .github/workflows/test.yaml (hunk 2 above)
Comment (steegecs) [reply .github/workflows/test.yaml]: Removed

**Pull request merged**